docs/pull_request.md

```diff
@@ -0,0 +1,56 @@
+# Pull Request
+
+To contribute for CloudCV in EvalAI is a facile task. Please follow these steps in order to get started :-)
+
+**Step 1 : Fork**
+
+1. Fork the EvalAI repository from [EvalAI](https://github.com/Cloud-CV/EvalAI) link.
+
+**Step 2 : Picking up a issue**
+
+1. Pick up a suitable issue that you find will be facile for you to fix. Moreover, you can also 
+take the issues based on their labels. Almost every issue is marked with a label according to its difficulty.
+
+2. After selecting any issue, mention any mentor’s name there and ask him to assign it to you.
```

Comment:
Replace with `After selecting any issue, ask the maintainers of the project to assign it to you and they will assign it based on its availability.`